Source/Microsoft.Teams.Apps.RemoteSupport.Configuration/ClientApp/src/components/input-text-form.tsx

```diff
@@ -31,7 +31,7 @@ const InputTextForm: React.FunctionComponent<IPropertiesProps> = (props) => {
                     <Input fluid placeholder={props.resourceStrings.placeholderText} value={properties.placeholder} onChange={(e: any) => { setProperties({ ...properties, placeholder: e.target.value }) }} />
                 </>
             </Flex.Item>
-            <Button content="Add Component" onClick={OnAddComponent} />
+            <Button content="Add Component" onClick={onAddComponent} />
```

Comment:
"Add Component" is a user-facing string, no?
Also it should be cased "Add component"

Reply:
Looks like you have this: props.resourceStrings.common.btnAddComponent

Reply:
replaced it with resourcestrings